docs/sections/cn_workflows.md

```diff
@@ -385,7 +385,22 @@ use the `--resource-manager` or `-r` option of the `popper run` subcommand to sp
 popper run -f sample.yml -r kubernetes
 ```
 
-For a workflow we build an image and create a volume in the namespace defined in the kubefig. This volume is the with a ReadWrite access.
+For a workflow we build an image and create a volume in the namespace defined in the [kubeconfig](https://kubernetes.io/docs/concepts/configuration/organize-cluster-access-kubeconfig/). 
+A sample kubeconfig looks like this:
```

Comment:
we can skip explaining what a `kubconfig` file is and instead only mention that the kubernetes runner makes use of the same authentication mechanism of `kubectl`, which is via the `kubeconfig` file (and then we add a link to the kubernetes documentation where this is explained)

Reply:
Noted, I'd make adequate changes